source/pages/albedo.ipynb

```diff
@@ -0,0 +1,358 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    pvlib.iotools.psm3.PSM_URL = "https://developer.nrel.gov/api/nsrdb/v2/solar/psm3-2-2-download.csv"
</pre><p>Perhaps add a comment as to why you are changing the base URL?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AssessingSolar/unofficial-psm3-userguide/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>